### ISSUE TITLE
Missing config.files in webUI

### ISSUE THREAD

**Issue by Allram** [guydavis/machinaris#186]
**Describe the bug**
The Machinaris webUI is not showing the config files for alerts, farming, plotting with the latest Test-branch docker-release. Previous release it did that. Machinaris works with the great config as long as i don't press Save on the WebUI.

**To Reproduce**

Steps to reproduce the behavior, e.g.:
1. Run latest Test branch (from 16/7-21)
2. Open Machinaris config in Webui
3. See error

**Expected behavior**
Config files to show like previous versions.

**System setup:**
 - OS: UnRaid
 - Docker version number: 20.10.5
 - Machinaris branch: test
 - Machinaris version number: 0.5.1

</details>

**Additional context & screenshots**
![image](https://user-images.githubusercontent.com/5573149/125903587-e5310c53-1673-494f-99bd-ffde469bfd5a.png)

**Comment by guydavis** [comment]
D'oh. Sorry!  Thanks for the report.  This was caused by script re-orderings on pages when I added table pagination last night.  

I've just patched this issue.  Also, included the [disabling of Chiadog notifications for partial proofs](https://github.com/martomi/chiadog/issues/252#issuecomment-877416135) that you suggested.  

Both fixes are now available in latest `:test` image.  Pull it down and try it out.  Thanks again for the feedback.
